### ISSUE TITLE
Annotations should indicate their tags

### ISSUE THREAD

**Issue by sopoforic** [burtonator/polar-bookshelf#1452]
On 2.0 beta 15, tagging a highlight works, but the highlight in the sidebar does not give any indication that it has been tagged. Perhaps the tags applied to an annotation should be displayed on it, but at least there should be some visual indicator--a colored tag icon, perhaps--that there are tags present.

In the annotations view, too, there is no way to see what tags an annotation has, short of clicking the tag button.

**Comment by burtonator** [comment]
Yes.  Agreed. We're working on adding this but not very happy with any of the designs Jonathan has been working on.  I think we're getting close.  The designs look good in Figma and I had experimented with turning them on.  I think we're going to implement his current design as he might be right that we've fixed the issue.  I was concerned that it was too noisy for the sidebar and made the UI look clunky so we were looking for an alternative.
